.github/workflows/codeql-codescan.yml

```
@@ -32,13 +32,13 @@
          uses: actions/checkout@v4
 
        - name: Initialize CodeQL
-         uses: github/codeql-action/init@v2
+         uses: github/codeql-action/init@v3
          with:
           languages: ${{ matrix.language }}
```

yamllint

[warning] wrong indentation: expected 11 but found 10

(indentation)